### ISSUE TITLE
Typo in TCP6ClientEndpoint._nameResolution's docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6327]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#6327|
|Type|defect|
|Created|2013-02-24 16:35:56Z|
|Branch|https://github.com/twisted/twisted/tree/fix-tcp6clientendpoint-docstring-6327|

"a tuple containig" should read "a tuple containing"

<details><summary>Searchable metadata</summary>

```
trac-id__6327 6327
type__defect defect
reporter__ashfall ashfall
priority__normal normal
milestone__ 
branch__branches_fix_tcp6clientendpoint_docstring_6327 branches/fix-tcp6clientendpoint-docstring-6327
branch_author__ashfall ashfall
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1361723756000000 1361723756000000
changetime__1361812236000000 1361812236000000
version__None None
owner__gen3 gen3

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_1"></a>|@ashfall commented|
|-|-|

(In [37239]) Branching to 'fix-tcp6clientendpoint-docstring-6327'

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @ashfall|
|-|-|

Thanks, please merge.

**Comment by twisted-trac** [comment]
|<img alt="gen3's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gen3 set owner to gen3<br>gen3 set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_5"></a>|@ashfall set status to `closed`|
|-|-|

(In [37252]) Merge fix-tcp6clientendpoint-docstring-6327: Fixing a typo in TCP6ClientEndpoint._nameResolution's docstring

Author: ashfall
Reviewer: thijs
Fixes: [#6327](https://github.com/twisted/twisted/issues/6327)
